api/app/signals/apps/reporting/csv/datawarehouse/tasks.py

```diff
@@ -32,25 +32,48 @@ def save_csv_file_datawarehouse(func: Callable[[str], str]) -> None:
             pass
 
         # Store the CSV files to the correct location
-        save_csv_files(csv_files=csv_files, using='datawarehouse')
+        return save_csv_files(csv_files=csv_files, using=using)
```

Comment:
Update the docstring with what this function returns (or remove the `:return:`).

Reply:
updated